coral-component-tablist/src/scripts/Tab.js

```diff
@@ -131,6 +132,12 @@ class Tab extends BaseLabellable(BaseComponent(HTMLElement)) {
     
     this.classList.toggle('is-invalid', this._invalid);
     this.setAttribute('aria-invalid', this._invalid);
+    if (this._invalid) {
+      this._elements.invalidIcon.removeAttribute('hidden');
```

Comment:
you can also use the property `hidden` so you can write `this._elements.invalidIcon = !this._invalid;`

Reply:
I've tested this and when `hidden='false'` I have observed a case where the icon is still hidden. I don't know the exact cause, probably due to the browser checking the existence of a hidden attribute, rather than the value of said attribute. I've added the `removeAttribute` specifically to counteract this and to not introduce any flakiness.

Reply:
`hidden='false'` is not the same as `hidden=false` :)

Reply:
Sorry, it was a typo. What I mean is that I tried `this._elements.invalidIcon = !this._invalid;` and got unexpected results.